Resources/templates/responsive/impact_discover/partials/projects_third_row.php

```diff
@@ -0,0 +1,26 @@
+<?php
+    $projects = $this->projects
+?>
+<div class="row">
+    <div class="col col-xs-12 col-sm-4">
+        <div class="row">
+            <?php if(current($projects)): ?>
+                <div href="/project/<?= current($projects)->id ?>" class="col col-xs-12 col-sm-12">
+                    <?= $this->insert('impact_discover/partials/project.php', ['project' => current($projects)]) ?>
+            </div>
+            <?php endif; ?>
+        </div>
+        <div class="row">
+            <?php if(next($projects)): ?>
+                <div href="/project/<?= current($projects)->id ?>" class="col col-xs-12 col-sm-12">
+                    <?= $this->insert('impact_discover/partials/project.php', ['project' => current($projects)]) ?>
+                </div>
+            <?php endif; ?>
+        </div>
+    </div>
+    <div href="/project/<?= current($projects)->id ?>" class="col col-xs-12 col-sm-8">
+        <?php if(next($projects)): ?>
+            <?= $this->insert('impact_discover/partials/project.php', ['project' => current($projects)]) ?>
+        <?php endif; ?>
+    </div>
+</div>
```

Comment:
@davidbeig couldn't these 3 templates be handled in just one template?

Reply:
Yes, they could, i just needed it to be in different places for clarity.